projects/kubernetes-sigs/cluster-api/patches/0005-Make-pause-and-bottlerocket-bootstrap-images-updatab.patch

```diff
@@ -31,18 +31,19 @@ index 9d5e58664..c763f6906 100644
  		{spec, kubeadmConfigSpec, clusterConfiguration, controllerManager, "*"},
  		{spec, kubeadmConfigSpec, clusterConfiguration, scheduler, "*"},
  		{spec, kubeadmConfigSpec, initConfiguration, nodeRegistration, "*"},
+ 		{spec, kubeadmConfigSpec, initConfiguration, patches, directory},
 +		{spec, kubeadmConfigSpec, joinConfiguration, "bottlerocketBootstrap", "*"},
 +		{spec, kubeadmConfigSpec, joinConfiguration, "pause", "*"},
  		{spec, kubeadmConfigSpec, joinConfiguration, nodeRegistration, "*"},
+ 		{spec, kubeadmConfigSpec, joinConfiguration, patches, directory},
  		{spec, kubeadmConfigSpec, preKubeadmCommands},
- 		{spec, kubeadmConfigSpec, postKubeadmCommands},
 diff --git a/controlplane/kubeadm/api/v1beta1/kubeadm_control_plane_webhook_test.go b/controlplane/kubeadm/api/v1beta1/kubeadm_control_plane_webhook_test.go
-index 9923c9527..62dbf938e 100644
+index 4d111c6b3..629385c6d 100644
 --- a/controlplane/kubeadm/api/v1beta1/kubeadm_control_plane_webhook_test.go
 +++ b/controlplane/kubeadm/api/v1beta1/kubeadm_control_plane_webhook_test.go
-@@ -594,6 +594,18 @@ func TestKubeadmControlPlaneValidateUpdate(t *testing.T) {
- 	validIgnitionConfigurationAfter := validIgnitionConfigurationBefore.DeepCopy()
```

Comment:
This logic changed to the below?

Reply:
This is probably the context lines too

Reply:
Same as above, they added a few lines below this in [this commit](https://github.com/kubernetes-sigs/cluster-api/commit/0146e81d7c51722de3d74d794cc948c75d959aaf), so we got new context lines.